.env

```
@@ -1,4 +1,6 @@
 COMPOSE_PROJECT_NAME=sentry-self-hosted
+SENTRY_SELF_HOSTED_ERRORS_ONLY=false
+COMPOSE_PROFILES=feature-complete
```

```suggestion
COMPOSE_PROFILES=sentry-feature-complete
```
Then get rid of `SENTRY_SELF_HOSTED_ERRORS_ONLY`, rename the profile to `sentry-feature-complete` in the compose.yml, and check `COMPOSE_PROFILES=sentry-feature-complete` in `...conf.py`. As it stands, we have two booleans representing one state ("is this errors only or not?"). We should avoid this redundant flagging, and just use the single boolean if possible.

---

done